backend/sendmessage/src/send-message-to-connection.ts

```diff
@@ -16,13 +16,10 @@ export const sendMessageToConnection = async (
       })
       .promise();
   } catch (e) {
-    if (!(e instanceof Error)) {
-      return;
-    }
-    if ((e as AWSError).statusCode === 410) {
+    if (e && (e as AWSError).statusCode === 410) {
       return removeConnection(config);
     } else {
-      throw e;
+      console.error(`Unexpected error when posting message to connection ${connectionId}:`, e);
```

Comment:
maybe i dont got it but why are we not rethrowing here?

Reply:
I added a comment to explain the reasoning better